src/api/print.api.js

```diff
@@ -134,10 +134,7 @@ class GeoAdminCustomizer extends BaseCustomizer {
                   )
                 : 0
             // don't ask why it works, but that's the best I could do.
-
-            symbolizer.graphicYOffset = symbolizer.graphicYOffset
-                ? (symbolizer.graphicYOffset = adjustWidth(-size[1], this.printResolution))
-                : 0
+            symbolizer.graphicYOffset = Math.round(symbolizer.graphicYOffset) ?? 0
```

Comment:
this won't work : 
Math.round(undefined) gives 'NaN', which satisfies the null operand and so it gives NaN instead of 0.
Also I would prefer we round it a little less, here is my proposal : 
```
symbolizer.gaphicYOffset = symbolizer.gaphicYOffset ?? 0
symbolizer.gaphicYOffset = Math.round(1000 * symbolizer.gaphicYOffset) / 1000
```
that way, we have a very insignificant rounding which still satisfies the current print at all times.

(we can go with a standard ternary operator instead if you prefer this approach)

Reply:
ah ok, thanks!